tests/functional/kata-deploy/kata-deploy.bats

```diff
@@ -73,6 +73,19 @@ setup() {
 	do
 		kubectl get runtimeclass | grep -E "${handler_re}"
 	done
+
+	# Ensure that kata-deploy didn't corrupt containerd config, by trying to get the container runtime and node status
+	echo "::group::kubectl pods"
+	kubectl get node -o wide
+	kubectl describe node --selector='!node-role.kubernetes.io/control-plane'
+	echo "::endgroup::"
+
+	ready_status=$(kubectl get node --no-headers --selector='!node-role.kubernetes.io/control-plane' -o custom-columns=STAUS:.status.conditions\[\?\(\@.type==\'Ready\'\)\].status)
```

Comment:
Even if it isn't printed because of `--no-headers`, `s/STAUS/STATUS` :smile:

Could there be a situation where `kata-deploy` would cause the node to be temporary not ready ?

If so, maybe do `kubectl wait node --selector='!node-role.kubernetes.io/control-plane' --for condition=Ready=True` ?

Reply:
Good suggestion - I think we also have the other way around, where it takes kata-deploy a little while to break the containerd config, so I'm not sure how best to balance them, but doing something like `kubectl wait node --selector='!node-role.kubernetes.io/control-plane' --timeout=60s --for condition=Ready=True` would give it a fair chance of succeeding, whether we want to bias in favour of success, or not in this test is a good question though!

Reply:
I've pushed this change - if nothing else it's easier to read and understand.

Reply:
Ah, slight issue that a bunch of the test clusters are only control-plane clusters without workers it seems so `kubectl wait node --selector='!node-role.kubernetes.io/control-plane' --timeout=60s --for condition=Ready=True` fails with `error: no matching resources found`, so we might need to re-think this approach

Reply:
Maybe `kubectl wait nodes --timeout=60s --all --for condition=Ready=True` is the play to check that all the node are ready?